app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -102,19 +102,52 @@ def default_authentication_type
     :bearer
   end
 
-  def scan_job_create(entity_class, entity_id)
+  def scan_job_create(entity)
+    check_policy_prevent(:request_containerimage_scan, :raw_scan_job_create, entity)
```

Comment:
I don't see any change in serialization done here? 
Take a look at raw_scan_job_create. The exact same parameters should be placed on the queue after this change. Maybe you mean somewhere else?